libraries/AP_Camera/AP_Camera_Backend.h

```diff
@@ -95,7 +102,10 @@ class AP_Camera_Backend
     float vertical_fov() const { return MAX(0, _params.vfov); }
 
     // handle MAVLink messages from the camera
-    virtual void handle_message(mavlink_channel_t chan, const mavlink_message_t &msg) {}
```

Comment:
Let's leave this handle_message() as virtual but then the MAVlinkCamV2 backend should call the parent handle_message() as either the first thing or last thing it does.  e.g call AP_Camera_Backend::handle_message()

Reply:
maybe add a comment to make it clear that all child classes, if they implement this method, must still call the parent's handle_message() method.